src/oop/programs_with_multiple_classes/hard/q4/QuestionText.md

```diff
@@ -0,0 +1,34 @@
+# oop-programs_with_multiple_classes-hard-q3
+
+You want to write a program that models books in a bookcase. To do this, create a Java program with two
+classes: `Book` and `Bookcase`. See the requirements below for each class.
+
+## Book Class
+
+Your `Book` class should have the following items:
+
+- Attributes:
+    - An `int` called **year**
+    - A `String` called **type**
+- Constructors:
+    - A constructor which accepts a String to initialize the **type** field and an int to initialize the **year**
+      field (in this order)
+        - This constructor should be accessible from other classes
+- Methods:
+    - A method called `determineTime`
+        - It should take no input and return an int
+        - Its purpose is to calculate the amount of years since the **year** when a book was
```

Comment:
This definitely needs a hint for the Year.now() part because I did not know that was even a thing

Reply:
The idea is that students wouldn't need to know this, they could just use "2023" or whatever the current year is. This will only be used in the test code, and it is in my solution code so the tests will continue to pass in coming years

Reply:
Ah ok

Reply:
Maybe we can specify to use the current year though?